lib/bibliothecary/parsers/go.rb

```diff
@@ -95,6 +105,39 @@ def self.parse_dep_lockfile(file_contents)
         map_dependencies(manifest, 'projects', 'name', 'revision', 'runtime')
       end
 
+      def self.parse_go_mod(file_contents)
+        deps = []
+        file_contents.lines.map(&:strip).each do |line|
+          next if line.start_with?("module ")
```

Comment:
Any interest in regex capturing this?
```
lines = file_contents.match(/require\s+\(\s?(.*)\)/m).captures[0]
```

Reply:
I'll just put this here before someone beats me to it.
<img width="567" alt="Screen Shot 2019-05-06 at 9 24 18 AM" src="https://user-images.githubusercontent.com/174416/57228019-c762a900-6fe0-11e9-86bb-42d92b26eccc.png">